tests/acceptance/environment.py

```diff
@@ -7,6 +7,14 @@ def before_feature(context, feature):
     if feature.name == 'Fixture loading':
         context.fixtures = ['behave-fixtures.json']
 
+    elif feature.name == 'Fixture loading with decorator':
+        # Including empty fixture to test that #92 is fixed
+        context.fixtures = ['empty-fixture.json']
+
+    elif hasattr(context, 'fixtures'):
+        # Removing fixtures defined above
+        del context.fixtures
```

Comment:
I don't understand why you're doing this. Do you suggest that everyone should do something like that? (Then this should be reflected in the documentation, by the way.)

I believe the tests should reflect real-world scenarios, the sunshine scenarios that should work. The last `elif`  block feels like you want to fix the tests. (Maybe I'm just too tired to understand.)

Reply:
This code isn't needed for the existing tests to pass, so I can remove.  I more added it to future proof that a fixture set in the previous few lines did not cause issues with other fixtures.

In a real world scenario, this is one way fixtures can be applied to specific features without them applying to other features, so this might be a good thing to mention in the documentation.